docs/src/further_topics/ugrid/data_model.rst

```diff
@@ -0,0 +1,565 @@
+  | Includes:
+
+  * | :attr:`~iris.experimental.ugrid.Mesh.topology_dimension`
+    | The maximum dimensionality of shape (1D=edge, 2D=face) supported
+      by this :class:`~iris.experimental.ugrid.Mesh`. Determines which
+      :class:`~iris.experimental.ugrid.Connectivity`\s are required/optional
+      (see below).
+
+  * 1-3 collections of :class:`iris.coords.AuxCoord`\s:
+
+    * | **Required**: :attr:`~iris.experimental.ugrid.Mesh.node_coords`
+      | The nodes that are the basis for the mesh.
+    * | Optional: :attr:`~iris.experimental.ugrid.Mesh.edge_coords`,
+        :attr:`~iris.experimental.ugrid.Mesh.face_coords`
+      | For indicating the 'centres' of the edges/faces.
+      | **NOTE:** generating a :class:`~iris.experimental.ugrid.MeshCoord` from
+        a :class:`~iris.experimental.ugrid.Mesh` currently (``Jan 2022``)
+        requires centre coordinates for the given ``location``; to be rectified
+        in future.
+
+  * 1 or more :class:`iris.experimental.ugrid.Connectivity`\s:
+
+    * | **Required for 1D (edge) elements**:
+        :attr:`~iris.experimental.ugrid.Mesh.edge_node_connectivity`
+      | Define the edges by connecting nodes.
+    * | **Required for 2D (face) elements**:
+        :attr:`~iris.experimental.ugrid.Mesh.face_node_connectivity`
+      | Define the faces by connecting nodes.
+    * Optional: any other connectivity type. See
+      :attr:`iris.experimental.ugrid.mesh.Connectivity.UGRID_CF_ROLES` for the
+      full list of types.
+
+.. doctest:: ugrid_summaries
+
+        >>> print(edge_cube.mesh)
+        Mesh : 'my_mesh'
+            topology_dimension: 2
+            node
+                node_dimension: 'Mesh2d_node'
+                node coordinates
+                    <AuxCoord: longitude / (degrees_east)  [...]  shape(5,)>
+                    <AuxCoord: latitude / (degrees_north)  [...]  shape(5,)>
+            edge
+                edge_dimension: 'Mesh2d_edge'
+                edge_node_connectivity: <Connectivity: unknown / (unknown)  [...]  shape(6, 2)>
+                edge coordinates
+                    <AuxCoord: longitude / (degrees_east)  [...]  shape(6,)>
+                    <AuxCoord: latitude / (degrees_north)  [...]  shape(6,)>
+            face
+                face_dimension: 'Mesh2d_face'
+                face_node_connectivity: <Connectivity: unknown / (unknown)  [...]  shape(2, 4)>
+                face coordinates
+                    <AuxCoord: longitude / (degrees_east)  [...]  shape(2,)>
+                    <AuxCoord: latitude / (degrees_north)  [...]  shape(2,)>
+            long_name: 'my_mesh'
+
+* | :class:`iris.experimental.ugrid.MeshCoord`
+  | Described in detail in `MeshCoords`_.
+  | Stores the following information:
+
+    * | :attr:`~iris.experimental.ugrid.MeshCoord.mesh`
+      | The :class:`~iris.experimental.ugrid.Mesh` associated with this
+        :class:`~iris.experimental.ugrid.MeshCoord`. Mirrored by the
```

Comment:
Might it be clearer to say
In place of "Mirrored by " ... the  /attr/ attribute of any Cube this is attached to.
Could have
 "This becomes"... or "This defines" ... or "This determines"

Reply:
8bc8c8488f912d10662f26edf94bff6c315df0e0